kalite/caching/__init__.py

```diff
@@ -1,99 +0,0 @@
-"""
```

Comment:
We now use neither the previous web page caching nor the in memory caches.

We should be using Django's in built caching mechanism for any caching in future.